### PR TITLE
Added a readme file for developing the UI

### PR DESCRIPTION
**Pull request by kaleb2**

### CODE REVIEW

File: ui/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# UI Development
+
+## Developer Tools
+
+Donwload and install the latest community edition of Qt Creator from here: https://www.qt.io/offline-installers
+
+a few install notes:    
+
+* The installer requires a free Qt account
+* Take care to claim pindividual ersonal use, and not enterprise
```
Comment (per-sephone) [inline ui/README.md]: typo here: "individual personal"

File: ui/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+* The installer requires a free Qt account
+* Take care to claim pindividual ersonal use, and not enterprise
+
+Reference documentation for the Qt Creator is here: https://doc.qt.io/qtcreator/
+
+And a sub section widget based apps such as this one is here: https://doc.qt.io/qtcreator/creator-using-qt-designer.html
+
+## Developer Guide
+
+First, do not manually edit the form.ui file in Qt Creator or anyhwere else. This is a automatically generated file based on the design in Qt Creator and should not be changed outside that context.
```
Comment (per-sephone) [inline ui/README.md]: typo here: "anywhere"

**Review by per-sephone — APPROVED** [review]
Just a couple typos (see individual comments), otherwise looks great!

**Review by jasonandmonte — APPROVED** [review]
Everything looks great!

I followed the directions and was able to make a small change to the form.ui and have it show up when I launched the program.

**Pull request merged**